aspnetcore/mvc/views/working-with-forms.md

```diff
@@ -184,7 +184,7 @@ Generates the following:
 
 With collection properties, `asp-for="CollectionProperty[23].Member"` generates the same name as `asp-for="CollectionProperty[i].Member"` when `i` has the value `23`.
 
-The calculated value of a `ModelExpression` is the first non-null result found in the ModelState entry with full name, the ViewData entry with full name, or the expression evaluated against Model, in that order. 
+Calculating the value of a `ModelExpression` looks at multiple sources including `ModelState` and `ViewData`. For instance, for `<input type="text" asp-for="@Name" />`, the calculated value is the first non-null result found in the ModelState entry with full name, the ViewData entry with full name, or the expression evaluated against Model, in that order. 
```

Comment:
1. Keep the back ticks around `ModelState`, `ViewData` and `Model`
2. To clarify the sentence, use more of the specific example e.g. "`ModelState` entry with name `"Name"` (or the full name if this is not a top-level node)" &hellip; "or the result of the expression `Model.Name`."